examples/use-cases/web_voyager_agent.py

```diff
@@ -56,13 +58,13 @@
 Find and book a hotel in Paris with suitable accommodations for a family of four (two adults and two children) offering free cancellation for the dates of February 14-21, 2025. on https://www.booking.com/
 """
 
-
 async def main():
 	agent = Agent(
 		task=TASK,
 		llm=llm,
 		browser=browser,
 		validate_output=True,
+		enable_memory=False,
```

Comment:
I think you are getting the Mem0 error since using an older version of Mem0 probably. Please use "mem0ai==0.1.88" or later.